src/OpenTelemetry/Trace/ActivitySourceAdapter.cs

```diff
@@ -107,6 +107,10 @@ private void RunGetRequestedData(Activity activity)
                 activity.Links);
 
             var samplingResult = this.sampler.ShouldSample(samplingParameters);
+            foreach (var attr in samplingResult.Attributes)
```

Comment:
Is this something OTel is planning to do or we will need to create an issue

Reply:
it's being addressed in otel